src/components/MutationsTable.tsx

```diff
@@ -0,0 +1,257 @@
+};
+
+export const BaselineMutationsTable = ({
+  baselineData,
+  mutations,
+  mutationType,
+  locationSelector,
+  baselineVariant,
+  dateRangeSelector,
+}: BaselineMutationsTableProps) => {
+  const headers: string[] = ['Additional', 'Missing'];
+  console.log('baseline data', baselineData);
+
+  return (
+    <>
+      <div className='mb-4 italic'>
+        *The table compares the mutation set of the variants in the collection to the fixed /{'>'}90%
+        mutations of the baseline.
+      </div>
+      <div className='table-container'>
+        <table className='mutationCustomTable' data-testid='baseline-mutation-table'>
+          <thead>
+            <tr>
+              <td> </td>
+              {headers.map(header => (
+                <td key={header} colSpan={4}>
+                  {header}
+                </td>
+              ))}
+            </tr>
+            <tr>
+              <td>Name</td>
+              {columnNames.map((name, i) => (
+                <td key={`${name}-${i}`}>{name}</td>
+              ))}
+              {columnNames.map((name, i) => (
+                <td key={`${name}/${i}`}>{name}</td>
+              ))}
+            </tr>
+          </thead>
+          <tbody>
+            {mutations &&
+              mutations.map(item => (
+                <tr key={item.name}>
+                  <td>
+                    {renderNameCell(
+                      item.query,
+                      locationSelector,
+                      item.name,
+                      true,
+                      dateRangeSelector,
+                      baselineVariant
+                    )}
+                  </td>
+                  {mutationColumnsProperties.map(field => {
+                    return (
+                      <td key={field.label}>
+                        {mutationType === 'aa'
+                          ? sortMutationList(
+                              'aa',
+                              pullAll(
+                                getNames(filterMutations(item.aaMutations, field)),
+                                getNames(filterBaselineMutations(baselineData.aaMutations))
+                              )
+                            ).join(', ')
+                          : sortMutationList(
+                              'nuc',
+                              pullAll(
+                                getNames(filterMutations(item.nucMutations, field)),
+                                getNames(filterBaselineMutations(baselineData.nucMutations))
+                              )
+                            ).join(', ')}
+                      </td>
+                    );
+                  })}
+
+                  {mutationColumnsProperties.map(field => {
+                    return (
+                      <td key={field.min}>
+                        {mutationType === 'aa'
+                          ? sortMutationList(
+                              'aa',
+                              compareToBaseline(
```

Comment:
What is this supposed to do? I don't quite get the meaning of "Missing" yet. Missing from what? What do the percentages mean (90% of what)?

The code selects mutations that are in the baseline and the complement of `field.min` in `item`. How does this connect to the headline "Missing"?

Reply:
The name of the function was a bit misleading, so I've renamed it to 'findMissingMutations'. It should find missing mutations based on this logic (as explained by @chaoran-chen): "If a fixed mutation of the baseline is present in 25% of variant samples - in other words missing in 75% of the samples, it should occur in the column 60-90% but in none of the others."

Reply:
Ok, makes sense. Please add a description to the table to clarify this. @chaoran-chen said he will provide a text at the beginning of next week.